api/diagnosis/diagnosis_routes.py

```
@@ -0,0 +1,88 @@
+        frame_count = 48
+        if len(resized_frames) < frame_count:
+            padding_frames = [np.zeros((145, 145, 3))] * (frame_count - len(resized_frames))
+            resized_frames.extend(padding_frames)
+        else:
+            resized_frames = resized_frames[:frame_count]
+
+        image_array = [frame / 255.0 for frame in resized_frames]
+        input_array = np.array(image_array)
+        input_array = np.expand_dims(input_array, axis=0)  # (1, 48, 145, 145, 3)
+
+        return input_array
```

Ruff (0.11.9)

Consider moving this statement to an `else` block

(TRY300)